plopfile.js

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable */
```

Comment:
Where individual files within the same directory need different eslint settings, prefer using the correct directives to blanket disabling eslint. In this case
```suggestion
/* eslint-env node */
```
which reveals some minor errors.